src/combinatorics/quiver/path.lean

```diff
@@ -59,6 +65,28 @@ def comp {a b : V} : Π {c}, path a b → path b c → path a c
 | c p q path.nil := rfl
 | d p q (path.cons r e) := by rw [comp_cons, comp_cons, comp_cons, comp_assoc]
 
+/-- Turn a path into a list. The list contains `a` at its head, but not `b` a priori. -/
+@[simp] def to_list : Π {b : V}, path a b → list V
```

Comment:
This list has the defining property that `list.chain (fun x y, nonempty (x ⟶ y))` or similar, right? If that's close to true, can you add the lemma to that effect?